qhub/template/{{ cookiecutter.repo_directory }}/image/Dockerfile.conda-store

```diff
@@ -0,0 +1,15 @@
+FROM quansight/conda-store:e2051a36e60bd3abd9aa44105f240b359ee6bab7
+
+RUN apt-get update \
```

Comment:
Just a thought, but might be worth considering adding the apt package `build-essential`, it includes a couple of compilers, make, and development libraries that are somewhat commonly needed when building packages.